src/app.rs

```diff
@@ -1443,8 +1472,6 @@ impl App {
                 '0' | '1' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9' => {
                     self.on_number(caught_char)
                 }
-                'u' => self.on_page_up(),
-                'd' => self.on_page_down(),
```

Comment:
Didn't even know we had this, nor was it ever documented as far as I'm aware.  I'm removing this for now, but will allow customizable keybinds in the future anyways.

Also it works with pgup/down now.